frontend/src/Settings/Notifications/Notifications/NotificationEventItems.js

```diff
@@ -83,6 +85,17 @@ function NotificationEventItems(props) {
             />
           </div>
 
+          <div>
+            <FormInputGroup
+              type={inputTypes.CHECK}
+              name="onDelete"
+              helpText="On Delete"
```

Comment:
Can we make this helpText a bit more informative (so users know its file delete and not movie delete)